custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/custom-metrics-prometheus-sd.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    run: custom-metric-prometheus-sd
+  name: custom-metric-prometheus-sd
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: custom-metric-prometheus-sd
+  template:
+    metadata:
+      labels:
+        run: custom-metric-prometheus-sd
+    spec:
+      containers:
+      - command:
+        - /bin/sh
+        - -c
+        - ./prometheus_dummy_exporter --metric-name=foo --metric-value=40 --port=8080
+        env:
+        - name: POD_ID
```

Comment:
Why set this env variable? It doesn't look like we're using it anywhere in this container.

Reply:
Done.